docs/en/ingest-management/commands.asciidoc

```diff
@@ -834,6 +835,63 @@ elastic-agent install --url=https://fleet-server:8220 \
   --certificate-authorities=/path/to/ca.crt
 ----
 
+++++
+<hr>
+++++
+
+[discrete]
+[[elastic-agent-otel-command]]
+== elastic-agent otel
+
+preview::[]
+
+Run {agent} as an <<otel-agent,OpenTelemetry Collector>>. 
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent otel [flags]
+elastic-agent otel [command]
+----
+
+[discrete]
+=== Available commands
+
+`--validate`::
```

Comment:
Not sure if I understand the markup correctly, but I think the command is `validate` without the dashes `--`?

For example:

```
./elastic-agent otel validate --config otel.yml
```

```suggestion
`validate`::
```

Reply:
Fixed!
![Screenshot 2024-06-18 at 1 29 05 PM](https://github.com/elastic/ingest-docs/assets/41695641/dbb4e156-7899-4a7d-90a3-4cb7c251e909)